src/xunit.performance.api/ScenarioConfiguration.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+
+namespace Microsoft.Xunit.Performance.Api
+{
+    /// <summary>
+    /// Provides an interface to configure how benchmark scenarios are run by the XunitPerformanceHarness.
+    /// </summary>
+    public sealed class ScenarioConfiguration
+    {
+        /// <summary>
+        /// Initializes a new instance of the ScenarioConfiguration class.
+        /// </summary>
+        /// <param name="milliseconds">The amount of time, in milliseconds, to wait for one iteration process to exit.</param>
+        /// <param name="iterations">Number of times a benchmark scenario process will be executed.</param>
+        public ScenarioConfiguration(double milliseconds, int iterations = 10)
+        {
+            if (milliseconds <= 0)
```

Comment:
I believe there is also a TimeSpan.Infinite constant, which we probably should support.